### PR TITLE
Ref#43689 category list

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43689](https://edu-redmine.sun-asterisk.vn/issues/43689)

## WHAT this PR does?
make list category

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

File: resources/views/admin/category/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<x-app-layout>
+    <x-slot name="slot">
+        <div class="mt-14">
+            <a href="{{ route('admin.categories.create') }}" class="bg-purple-500 hover:bg-purple-700 text-white text-center py-2 px-4 rounded">
+                {{ ('New') }}
```
Comment (huyenttm-0901) [inline resources/views/admin/category/index.blade.php]: i18n

**Comment by hoangtrungthong** [comment]
ready

**Comment by ghost** [comment]
mấy pull giao diện này sau thêm screenshot cho m nha

**Pull request merged**